File: Mainframe.BuyerSupplier.Core/Dto/UserCreationInitialDataDto.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Mainframe.BuyerSupplier.Core.Dto
{
    public class UserCreationInitialDataDto
    {
        public List<SupplierStandardInventoryDto> StandardInventoryList { get; set; }
        public List<SupplierBaseDto> SupplierBaseList { get; set; }
        public List<DeliverySlotsDto> DeliverySlotList { get; set; }
        public List<ZoneDto> ZoneList { get; set; }

    }
}

File: Mainframe.BuyerSupplier.Ui/admin-app/src/app/Users-List/UserCreationInitialDataDto.ts
import { StandardInventoryDto } from "../standard-inventory/StandardInventoryDto";
import { SupplierBaseDto } from "../supplierBaseManagement/supplierBaseDto";
import { DeliverySlotsDto } from "../delivery-slots/DeliverySlotsDto";
import { SupplierStandardInventoryDto } from "../user/SupplierStandardInventoryDto";
import { ZoneDto } from "../zone/ZoneDto";

export class UserCreationInitialDataDto
{
  standardInventoryList : SupplierStandardInventoryDto[];
  supplierBaseList : SupplierBaseDto[];
  deliverySlotList: DeliverySlotsDto[];
  zoneList: ZoneDto[];
}